### PR TITLE
fix: upload collection overflow [ci visual]

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue

Part of https://github.com/SAP/fundamental-ngx/issues/7815

Follow-up pr in ngx https://github.com/SAP/fundamental-ngx/pull/7857

## Description

Overflow handling improvements.
Modifier classes added to the upload collection item object status to handle overflow approriate.

### Before:

image

### After:

image

## BREAKING CHANGE:
* Upload collection item object status layout changed & now needs additional classes to be applied

Wiki https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes#upload-collection-3227

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
This is a Breaking Change. Please document it in the Wiki and in the PR description

**Comment by platon-rov** [comment]
> This is a Breaking Change. Please document it in the Wiki and in the PR description

Done. https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes#upload-collection-3227

File: src/styles/upload-collection.scss [hunk 1]
```diff
@@ -53,8 +54,25 @@ $fd-upload-collection-button-group-small-margin: 0.5rem !default;
...
+
+      flex-shrink: 1;
+      min-width: 0;
+
+      &:first-child {
+        @include fd-set-margins-x(0, 0.25rem);
+      }
+
+      &-text {
+        @include fd-ellipsis();
```
Comment (g-cheishvili) [inline src/styles/upload-collection.scss:73]:
```suggestion
        @include fd-ellipsis();
        line-height: initial;
```
This should fix that cut-down characters

**Review by g-cheishvili — CHANGES_REQUESTED** [review]
GG. Just take care of those massacred characters

**Review by aberikashvili — APPROVED** [review]
LGTM

**Comment by g-cheishvili** [comment]
For some reason action buttons are vertically aligned in case of long texts, but horizontally when short texts. Is that how it should be? @platon-rov

**Review by g-cheishvili — APPROVED** [review]

**Pull request merged**